petab/core.py

```diff
@@ -701,24 +781,9 @@ def get_placeholders(formula_string, observable_id, override_type):
     return placeholders
 
 
-def get_dynamic_parameters_from_sbml(sbml_model: libsbml.Model):
-    """
-    Get list of non-constant parameters in SBML model.
-    """
-    return [p.getId() for p in sbml_model.getListOfParameters()
-            if not p.getConstant()]
-
-
-def get_dynamic_simulation_parameters(sbml_model, parameter_df):
-    par_opt_ids = get_optimization_parameters(parameter_df)
-    sbml_pars = sbml_model.getListOfParameters()
-    par_ids = [
-        p.getId() for p in sbml_pars
-        if p.getId().startswith("noiseParameter")
-        or p.getId().startswith("observableParameter")
-        or p.getId() in par_opt_ids
-    ]
-    return par_ids
+def get_model_parameters(sbml_model: libsbml.Model):
+    """Return list of SBML model parameter IDs"""
+    return [p.getId() for p in sbml_model.getListOfParameters()]
```

Comment:
no getConstant() required any more? I.e. all parameters in the sbml are to be included here?

Reply:
Yes. This needs to be handled downstream. The constant thing was kind of a hack we used, but had nothing to do with the SBML's intention. This might cause trouble in other tools. Therefore, I would not introduce that here.